applications/GeoMechanicsApplication/custom_utilities/dof_utilities.cpp

```diff
@@ -0,0 +1,116 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Anne van de Graaf
+//
+
+#include <algorithm>
+
+#include "dof_utilities.h"
+#include "includes/variables.h"
+
+namespace Kratos
```

Comment:
To avoid conflicts with other applications, I'd propose to make this a separate namespace
```suggestion
namespace Kratos::DofUtilities
```

Reply:
Good suggestion. I have made the namespace even more specific: `Kratos::Geo::DofUtilities`.